pkgs/swift2objc/lib/src/transformer/transform.dart

```diff
@@ -14,6 +14,9 @@ import '_core/unique_namer.dart';
 import 'transformers/transform_compound.dart';
 import 'transformers/transform_globals.dart';
 
+// TODO: Update TransformationMap to make referencing types easy.
```

Comment:
TODOs should have a bug. Putting more info about this in a bug would also help me understand it and discuss solutions, because I'm not really sure what the issue is here.